pyresample/bilinear/xarr.py

```diff
@@ -28,7 +30,8 @@ def __init__(self,
                  nprocs=1,
                  segments=None):
         """
-        Parameters
+        Parameters.
+
```

Comment:
Extra blank line here after Parameters.

Reply:
Yes, that's what pre-commit and flake8 wanted. Wouldn't let me commit without it. The first line (the shoet description of the function/method) should end in a full stop, followed by an empty line.

Reply:
Oh it's because you don't have a description of the method. It should be:

```
"""Initialize resampler.

Parameters
-----------

```

What you have now is "Parameters." is the "subject line" of the method.

Reply:
Yeah. Had to do a quick modification to quite many places so could push the real changes before the day ended. I'll update that and check also the other bits on monday.